### PR TITLE
Fix anaconda build

### PR DESCRIPTION
**Pull request by fdiblen**
- conda package fixes
  - move conda receipe to conda folder
  - fix conda package installation issue
  - add extra import tests for conda package
  - add instructions to build conda package locally
  - automatically find matchms package in setup.py
  - update developer instructions
  - increase verbosity while packaging
  - skip builds for Python 2.X
  - more flexible package versions
  - add deployment requirements to meta.yml

### CODE REVIEW

File: conda/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  description: |
+    Python library for fuzzy comparison of mass spectrum data and other Python objects
+  doc_url: https://matchms.readthedocs.io/
+  dev_url: https://github.com/matchms/matchms
+
+extra:
+  deploy:
+    - anaconda-client
+    - setuptools >=0.39
+    - twine >=1.12
```
Comment (jspaaks) [inline conda/meta.yaml]: remove twine

**Review by jspaaks — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- conda package fixes https://github.com/matchms/matchms/pull/223
```
Comment (sverhoeven) [inline CHANGELOG.md]:
```suggestion
- conda package fixes [#223](https://github.com/matchms/matchms/pull/223)
```

File: README.rst [hunk 3]
```diff
@@ -95,7 +95,7 @@ Install matchms from Anaconda Cloud with
 
 .. code-block:: console
 
-  conda install --channel nlesc matchms
+  conda install -c nlesc -c bioconda -c conda-forge matchms
```
Comment (sverhoeven) [inline README.rst]:
```suggestion
  conda install --channel nlesc --channel bioconda --channel conda-forge matchms
```

File: conda/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    - pyteomics >=4.2
+    - python >=3.7,<3.8
+    - pyyaml
+    - rdkit >=2020.03.1
+    - scipy
+
+test:
+  imports:
+    - matchms
+    - matchms.exporting
```
Comment (sverhoeven) [inline conda/meta.yaml]: Please remove all nested packages, they are already imported by `import matchms`

File: setup.py [hunk 5]
```diff
@@ -1,43 +1,49 @@
...
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: Apache Software License",
+        "Natural Language :: English",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.7"
     ],
-    test_suite='tests',
+    test_suite="tests",
     install_requires=[
         # see environment.yml
```
Comment (sverhoeven) [inline setup.py]:
```suggestion
        # see conda/environment.yml
```

File: conda/meta.yaml (hunk 4 above)
Comment (sverhoeven) [reply conda/meta.yaml]: Fixed in https://github.com/matchms/matchms/pull/223/commits/dd0d24c545caab97b6db0fce3194e03d593af686

**Review by sverhoeven — APPROVED** [review]

**Pull request merged**